src/foam/u2/FloatView.js

```diff
@@ -51,22 +64,22 @@ foam.CLASS({
       this.SUPER(p);
 
       this.precision = p.precision;
-      this.min       = p.min;
-      this.max       = p.max;
+      this.min = p.min;
+      this.max = p.max;
     },
 
     function formatNumber(val) {
       if ( ! val ) return '0';
       val = val.toFixed(this.precision);
       var i = val.length - 1;
-      for ( ; i > 0 && val.charAt(i) === '0' ; i-- ) {}
+      for ( ; i > 0 && val.charAt(i) === '0'; i -- ) {}
```

Comment:
The space between i and -- doesn't comply with the google/foam style guideline.

Reply:
You're right. Since we enforce a space between `!` and its argument as part of our rule exceptions, our linter flags all unary operators without a space between them and their arguments. I'll revert these spots back and have a quick look to see if the linter settings can be more fine-grained.